packages/api/src/command/medical/document/document-query.ts

```diff
@@ -104,6 +119,7 @@ export async function updateDocQuery(params: UpdateDocQueryParams): Promise<Pati
 export const updateConversionProgress = async ({
   patient,
   convertResult,
+  requestId,
```

Comment:
I'm thinking we shouldn't get the `requestId` here, only on `appendDocQueryProgress` - which is the function that gets called to set doc query progress. This one (`updateConversionProgress`) should be used to update the status of conversion/download while in progress, IIRC.

I think this would also simplify this PR, minimizing updates.